scaleway/resource_rdb_instance_beta.go

```diff
@@ -253,7 +268,26 @@ func resourceScalewayRdbInstanceBetaUpdate(d *schema.ResourceData, m interface{}
 		if err != nil {
 			return err
 		}
+
+		// Wait for the instance to settle after upgrading
+		time.Sleep(30 * time.Second)
```

Comment:
What issue did you encounter?

Reply:
This is an issue on RDB side. I will remove this when it is resolved.